src/frontend/src/common/redux/utils/groupUtils.ts

```diff
@@ -14,7 +14,15 @@ export const ensureValidGroup = async (): Promise<void> => {
 
   const { dispatch, getState } = store;
   const state = getState();
-  const rawUserInfo = await fetchUserInfo();
+  let rawUserInfo: RawUserRequest | null = null;
+  try {
```

Comment:
I'm surprised about needing the try/catch here around the `fetchUserInfo`. It probably shouldn't bubble up an error even if something goes wrong. Is this the outstanding problem for non-logged-in users on the homepage where the requests get swatted down b/c no auth, so we have errors because of that, or is it something else?

I do think we should deal with that at some point, but I feel like it should happen more inside the level of the react-query `fetchUserInfo` or sub-functions doing the actual request than up here at the call site. Maybe we could just make an eng excellence ticket to deal with these kinds of expected auth "errors" when not logged in and hopefully we'll get to it in the future?

Reply:
This is a personal patch I've been using to deal with the frontend app erroring out because the userinfo request failed for whatever reason, which happens to me all the time in localdev. I didn't actually mean to add it to this PR, I'll take it out